src/components/PostsList.tsx

```diff
@@ -1,86 +1,65 @@
-import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[];
+  isloadingComments?: boolean;
```

Comment:
The `isloadingComments` prop is defined but not used within the component. If it's not needed, consider removing it from the `Props` type definition to keep the code clean and maintainable.